server/tserver/src/main/java/org/apache/accumulo/tserver/log/LogSorter.java

```diff
@@ -242,7 +244,9 @@ public void startWatchingForRecoveryLogs(ThreadPoolExecutor distWorkQThreadPool)
         RecoveryStatus status = new RecoveryStatus();
         status.name = entries.getKey();
         try {
-          status.progress = entries.getValue().getBytesCopied() / (0.0 + conf.getAsBytes(Property.TSERV_WALOG_MAX_SIZE));
+          double progress = entries.getValue().getBytesCopied() / walBlockSize;
+          // to be sure progress does not exceed 100%
+          status.progress = Math.max(progress, 99.0);
```

Comment:
I think this keeps the status permanently at 99%. I think we want something like `Math.min(progress, 99.9)`